### PR TITLE
fix: windows path issue

### LINKED ISSUE #14: fix: absolute paths error on windows
The issue when you add @dargmuesli/nuxt-cookie-control to the modules section of nuxt.config.ts

Cannot start nuxt:  Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'        

  at new NodeError (node:internal/errors:371:5)
  at defaultResolve (node:internal/modules/esm/resolve:1016:11)
  at ESMLoader.resolve (node:internal/modules/esm/loader:422:30)
  at ESMLoader.getModuleJob (node:internal/modules/esm/loader:222:40)
  at ESMLoader.import (node:internal/modules/esm/loader:276:22)
  at importModuleDynamically (node:internal/modules/esm/translators:111:35)
  at importModuleDynamicallyCallback (node:internal/process/esm_loader:35:14)
  at loadLocales (/C:/Users/akubasa/workspace/nuxt-app/node_modules/@dargmuesli/nuxt-cookie-control/dist/module.mjs:175:31)
  at async setup (/C:/Users/akubasa/workspace/nuxt-app/node_modules/@dargmuesli/nuxt-cookie-control/dist/module.mjs:101:5)
  at async normalizedModule (/C:/Users/akubasa/workspace/nuxt-app/node_modules/@nuxt/kit/dist/index.mjs:167:5)
  at async installModule (/C:/Users/akubasa/workspace/nuxt-app/node_modules/@nuxt/kit/dist/index.mjs:416:3)
  at async initNuxt (/C:/Users/akubasa/workspace/nuxt-app/node_modules/nuxt/dist/index.mjs:1825:7)
  at async load (/C:/Users/akubasa/workspace/nuxt-app/node_modules/nuxi/dist/chunks/dev.mjs:6779:9)
  at async Object.invoke (/C:/Users/akubasa/workspace/nuxt-app/node_modules/nuxi/dist/chunks/dev.mjs:6840:5)
  at async _main (/C:/Users/akuba

### PR DESCRIPTION
**Pull request by zal1000**
fix #14 
Tested on win11, devcontainer

### CODE REVIEW

File: src/module.ts [hunk 1]
```diff
@@ -132,9 +132,9 @@ const loadLocales = async (moduleOptions: ModuleOptions) => {
   moduleOptions.locales = []
 
   for (const locale of locales) {
-    const text = await import(
-      await resolvePath(resolve(runtimeDir, 'locale', locale))
-    ).then((r: any) => r.default || r)
+    const rpath = await resolvePath(resolve(runtimeDir, "locale", locale))
+    const path = !rpath.toLowerCase().startsWith("file://") || !rpath.toLowerCase().startsWith("/") ? `file://${rpath}` : rpath
```
Comment (dargmuesli) [inline src/module.ts:136]: I'm curious, how did you come up with those changes, especially the `file://` prefix?

**Review by dargmuesli — CHANGES_REQUESTED** [review]
Thank you for this fix, @zal1000!

Please make sure to follow linting rules as well as our [commit guideline](https://www.conventionalcommits.org/en/v1.0.0/). Alternatively, I can fix those errors for you if that's ok for you.

**Comment by JulienChampagnol** [comment]
Hi, any update on this?

**Comment by dargmuesli** [comment]
Please follow the upstream issue for updates: https://github.com/nuxt/nuxt/issues/15500

**Comment by dargmuesli** [comment]
Expecting the upstream issue to result in a fix, which would make PRs on downstream repos like this one redundant. Therefore, I'm closing this PR. Thank you for your work!

**Pull request closed without merging**